src/visual/aria.ts

```diff
@@ -17,123 +17,186 @@
 * under the License.
 */
 
-// @ts-nocheck
-
 import * as zrUtil from 'zrender/src/core/util';
-import { retrieveRawValue }
-
-from '../data/helper/dataProvider';
-
-export default function (dom, ecModel) {
-    const ariaModel = ecModel.getModel('aria');
-    if (!ariaModel.get('show')) {
-        return;
-    }
-    else if (ariaModel.get('description')) {
-        dom.setAttribute('aria-label', ariaModel.get('description'));
+import ExtensionAPI from '../ExtensionAPI';
+import {retrieveRawValue} from '../data/helper/dataProvider';
+import GlobalModel from '../model/Global';
+import Model from '../model/Model';
+import {AriaOption} from '../component/aria';
+import {TitleOption} from '../component/title';
+
+export default function (ecModel: GlobalModel, api: ExtensionAPI) {
+    const ariaModel: Model<AriaOption> = ecModel.getModel('aria');
```

Comment:
A default option can be merged here.
```ts
ariaModel.option = isObject(ariaModel.option) ? defaults(ariaModel.option, defaultOption) : null;
```

And the `enabled` property can be default true.